### PR TITLE
feat(i18n): add zh-Hant translation

### PR DESCRIPTION
**Pull request by imgarylai**
Got some code for us? Awesome 🎊!

Please include a description of your change & check your PR against this list, thanks!

- [x] There's a clear use-case for this code change, explained below
- [x] Commit message has a short title & references relevant issues
- [x] The build will pass (run `yarn test:all` and `yarn lint`)

We appreciate your contribution!

### CODE REVIEW

**Review by cathysarisky — CHANGES_REQUESTED** [review]
Hello!  Thank you for submitting these translations!
I'm happy to see that you were able to submit the missing portal strings and the ones needed to make search translated.
I'm helping out by taking a quick look at translations, with some help from AI.  I left a few comments where I thought that maybe the translation needed another look.  Please feel welcome and encouraged to indicate that any of my questions are irrelevant by marking them as resolved or by leaving a response.
When you've had a chance to make any needed changes (or left a comment that none of my concerns require changes), please let me know so that I can change my review to approved.

File: ghost/i18n/locales/zh-Hant/portal.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "(save {{highestYearlyDiscount}}%)": "",
+    "(save {{highestYearlyDiscount}}%)": "省下 {{highestYearlyDiscount}}%",
```
Comment (dlackty) [inline ghost/i18n/locales/zh-Hant/portal.json:2]: Maybe we should keep `（）`？

File: ghost/i18n/locales/zh-Hant/portal.json [hunk 2]
```diff
@@ -56,8 +56,8 @@
     "Emails": "電子報",
     "Emails disabled": "已停止接收電子報",
     "Ends {{offerEndDate}}": "於 {{offerEndDate}} 結束",
-    "Enter your email address": "",
-    "Enter your name": "",
+    "Enter your email address": "你的 email",
```
Comment (dlackty) [inline ghost/i18n/locales/zh-Hant/portal.json:59]: Maybe `輸入你的 Email`？

File: ghost/i18n/locales/zh-Hant/portal.json [hunk 3]
```diff
@@ -56,8 +56,8 @@
...
+    "Enter your name": "你的名字",
```
Comment (dlackty) [inline ghost/i18n/locales/zh-Hant/portal.json:60]: Same, maybe `輸入你的名字`

File: ghost/i18n/locales/zh-Hant/portal.json [hunk 4]
```diff
@@ -77,15 +77,15 @@
     "If you've completed all these checks and you're still not receiving emails, you can reach out to get support by contacting {{supportAddress}}.": "如果您已經完成了上述所有檢查項目，但仍然沒有收到電子報，您可以透過聯繫 {{supportAddress}} 取得協助。",
     "In the event a permanent failure is received when attempting to send a newsletter, emails will be disabled on the account.": "要是系統發送電子報時遇到永久失敗的情形，該帳號將停止接收電子報。",
     "In your email client add {{senderEmail}} to your contacts list. This signals to your mail provider that emails sent from this address should be trusted.": "在您的 email 系統中將 {{senderEmail}} 加入您的聯絡人列表中。如此一來，您的 email 系統就會知道可以信任從該地址發送的郵件。",
-    "Invalid email address": "",
-    "Jamie Larson": "",
-    "jamie@example.com": "",
+    "Invalid email address": "email 有誤",
```
Comment (dlackty) [inline ghost/i18n/locales/zh-Hant/portal.json:80]: Maybe `不合法的 Email 地址` is better.

File: ghost/i18n/locales/zh-Hant/portal.json [hunk 5]
```diff
@@ -77,15 +77,15 @@
...
+    "Jamie Larson": "王小明",
```
[the remark this answers is not in the document]
Comment (dlackty) [reply ghost/i18n/locales/zh-Hant/portal.json:81]: Yes, I can confirm that. Actually there's a [wikipedia page](https://zh.wikipedia.org/zh-tw/%E7%8E%8B%E5%B0%8F%E6%98%8E) for this name as placeholder.

File: ghost/i18n/locales/zh-Hant/portal.json [hunk 6]
```diff
@@ -117,7 +117,7 @@
     "Sign out": "登出",
     "Sign up": "註冊",
     "Signup error: Invalid link": "註冊錯誤：連結無效",
-    "Something went wrong, please try again later.": "",
+    "Something went wrong, please try again later.": "伺服器錯誤，請重試。",
```
Comment (dlackty) [inline ghost/i18n/locales/zh-Hant/portal.json:120]: Maybe we should add "later" in translation: `伺服器錯誤，請稍後重試。`

File: ghost/i18n/locales/zh-Hant/portal.json [hunk 7]
```diff
@@ -132,26 +132,26 @@
...
-    "Thank you for your support": "",
-    "Thank you for your support!": "",
+    "Thank you for your support": "感謝你的支持",
+    "Thank you for your support!": "感謝你的支持！",
     "Thanks for the feedback!": "感謝您的意見！",
     "That didn't go to plan": "發生錯誤",
     "The email address we have for you is {{memberEmail}} — if that's not correct, you can update it in your <button>account settings area</button>.": "就我們所知，您的 email 地址是 {{memberEmail}}。如果有誤，您可以在<button>帳號設定區塊</button>進行更新。",
     "There was a problem submitting your feedback. Please try again a little later.": "提交您的意見時遇到問題。請稍後再試。",
-    "There was an error processing your payment. Please try again.": "",
+    "There was an error processing your payment. Please try again.": "付款失敗，麻煩你再試一次",
```
Comment (dlackty) [inline ghost/i18n/locales/zh-Hant/portal.json:141]: Maybe `付款發生錯誤，請你再試一次。`? Just want to consistently use 請 as please

File: ghost/i18n/locales/zh-Hant/portal.json [hunk 8]
```diff
@@ -132,26 +132,26 @@
...
     "This site is invite-only, contact the owner for access.": "此網站僅限受邀請者觀看，請聯繫網站擁有者取得存取權限。",
-    "This site is not accepting payments at the moment.": "",
+    "This site is not accepting payments at the moment.": "該付款方式尚未支援",
     "To complete signup, click the confirmation link in your inbox. If it doesn't arrive within 3 minutes, check your spam folder!": "要完成註冊，請點擊您收件匣中的確認連結。如果在 3 分鐘內沒有收到，請檢查您的垃圾郵件。",
-    "To continue to stay up to date, subscribe to {{publication}} below.": "",
+    "To continue to stay up to date, subscribe to {{publication}} below.": "繼續訂閱 {{publication}} 以獲得最新消息",
```
[the remark this answers is not in the document]
Comment (dlackty) [reply ghost/i18n/locales/zh-Hant/portal.json:145]: Yes, Cathy is right. I would advice `訂閱 {{publication}} 以持續獲得最新消息`

File: ghost/i18n/locales/zh-Hant/portal.json [hunk 9]
```diff
@@ -132,26 +132,26 @@
...
     "Try free for {{amount}} days, then {{originalPrice}}.": "免費試用 {{amount}} 天，然後以 {{originalPrice}} 開始訂閱。",
     "Unlock access to all newsletters by becoming a paid subscriber.": "成為付費會員以解鎖所有電子報內容。",
-    "Unsubscribe from all emails": "取消所有電子報訂閱",
+    "Unsubscribe from all emails": "取消訂閱所有電子報",
     "Unsubscribed": "未訂閱",
-    "Unsubscribed from all emails.": "",
+    "Unsubscribed from all emails.": "取消訂閱所有電子報",
```
[the remark this answers is not in the document]
Comment (dlackty) [reply ghost/i18n/locales/zh-Hant/portal.json:150]: Yes, this should be `未訂閱任何電子報` instead.

File: ghost/i18n/locales/zh-Hant/portal.json [hunk 10]
```diff
@@ -132,26 +132,26 @@
...
     "Unsubscribing from emails will not cancel your paid subscription to {{title}}": "取消接收電子報不會取消您對 {{title}} 的付費訂閱。",
     "Update": "更新",
     "Update your preferences": "更新您的偏好設定",
-    "Verification link sent, check your inbox": "",
+    "Verification link sent, check your inbox": "確認信件已經寄出，請查收",
```
Comment (dlackty) [inline ghost/i18n/locales/zh-Hant/portal.json:154]: May we use `驗證連結已寄出，請確認你的信箱` instead?

**Review by dlackty — COMMENTED** [review]
Hi, native zh-Hant speaker here :) I added few comments and I can help create a commit or new PR too if @imgarylai is busy.

Thanks @imgarylai & @cathysarisky !

**Comment by imgarylai** [comment]
Hey @cathysarisky @dlackty, thank you for those suggestion. I was on vacation last week. I will address these comments and publish another revision. 

Thanks

**Review by dlackty — APPROVED** [review]

**Review by cathysarisky — COMMENTED** [review]
Thanks for the additions! 

I notice that Github thinks you added some changes to casper, probably on accident?

**Comment by imgarylai** [comment]
There was some issue on git submodule sync. I syncced it and the problem has been resolved. Thanks!

**Review by cathysarisky — APPROVED** [review]

**Pull request merged**